app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -32,7 +32,7 @@ public static SessionStore get() {
         }
         return mInstance;
     }
-    private static final String HOME_WITHOUT_REGION_ORIGIN = "https://webxr.today";
+    public static final String HOME_WITHOUT_REGION_ORIGIN = "https://webxr.today";
```

Comment:
you can keep this `private` if you use the `isHomeUri` helper method, per my comment above